client/components/telecom/pack/migration/pack-migration-process.service.js

```diff
@@ -104,6 +104,14 @@ angular.module('managerApp').service('PackMigrationProcess', function ($q, OvhAp
       });
     }
 
+    // building details post params
+    postParams.buildingReference = migrationProcess.selectedOffer.buildingReference;
+    postParams.stair = migrationProcess.selectedOffer.stair;
+    postParams.floor = migrationProcess.selectedOffer.floor;
```

Comment:
You can also `_.assign(postParams, {
builidingReference: migrationOffer.selectedOffer.buildingReference,
stair: ...,
floor: ...
})`